test/e2e/util/velero/velero_utils.go

```diff
@@ -797,3 +807,29 @@ func GetResticRepositories(ctx context.Context, veleroNamespace, targetNamespace
 
 	return common.GetListBy2Pipes(ctx, *CmdLine1, *CmdLine2, *CmdLine3)
 }
+
+func GetSnapshotCheckPoint(client TestClient, VeleroCfg VerleroConfig, expectCount int, namespaceBackedUp, backupName string, kibishiiPodNameList []string) (SnapshotCheckPoint, error) {
+	var snapshotCheckPoint SnapshotCheckPoint
+	snapshotCheckPoint.ExpectCount = expectCount
+	snapshotCheckPoint.NamespaceBackedUp = namespaceBackedUp
+	snapshotCheckPoint.PodName = kibishiiPodNameList
+	if strings.EqualFold(VeleroCfg.Features, "EnableCSI") {
+		snapshotCheckPoint.EnableCSI = true
+		if err := util.CheckVolumeSnapshotCR(client, kibishiiPodNameList, namespaceBackedUp, backupName); err != nil {
+			return snapshotCheckPoint, errors.Wrapf(err, "Fail to get Azure CSI snapshot content")
+		}
+		for _, podName := range kibishiiPodNameList {
+			snapshotContentName, err := util.GetVolumeSnapshotContentNameByPod(client, podName, namespaceBackedUp, backupName)
+			if err != nil {
+				return snapshotCheckPoint, errors.Wrap(err, "Fail get VolumeSnapshotContent name by pod")
+			}
+			snapshotCheckPoint.SnapshotIDList = append(snapshotCheckPoint.SnapshotIDList, strings.Replace(snapshotContentName, "snapcontent", "snapshot", 1))
+			fmt.Println(snapshotCheckPoint)
+		}
+		if len(snapshotCheckPoint.SnapshotIDList) != expectCount {
```

Comment:
Except the count of VolumeSnapshots, also need to verify VolumeSnapshots' ReadyToUse in Status is set to true